client/ayon_unreal/plugins/load/load_camera.py

```diff
@@ -10,13 +10,15 @@
     get_representation_path,
 )
 from ayon_unreal.api import plugin
+
+from ayon_unreal.api.storage import  HALON_PATH_CONFIG
```

Comment:
I'm no so sure about getting this as a global. I know we did a lot of that in the HalonBasePlugin, but that was more because we didn't have a whole long of control at the time. I wonder if we would be better off getting the information locally in the module using context tools. I think by doing so it'll allow the settings up update without having to restart unreal.